lib/domain/GooglePay.js

```diff
@@ -0,0 +1,38 @@
+const Joi = require('joi');
+const { CaseInsensitiveSchema } = require('../schema');
+const constraints = require('./_constraints');
+
+const PaymentTransaction = require('./common/PaymentTransaction');
+
+/**
+ * Used to create a new payment in the Barion system.
+ *
+ * @see {@link https://docs.barion.com/Payment-Start-v2|Barion API Documentation}
+ */
+const schema = Joi.object({
```

Comment:
Docs refers this as an extension of the `StartPayment` schema, please use schema extension here: https://joi.dev/api/?v=17.7.0#objectkeysschema